Our.Umbraco.TagHelpers/Extensions/WebHostEnvironmentExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using System;
+using System.IO;
+using Umbraco.Cms.Core;
+
+namespace Our.Umbraco.TagHelpers.Extensions
+{
+    [Obsolete("This should be removed, when the package gets upgraded past Umbraco 10")]
+    internal static class WebHostEnvironmentExtensions
```

Comment:
@skttl so if this was only for Umbraco 11 then we would need this class?

Reply:
Yes :)